client/src/components/Bio/Bio.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from "react";
+import "./Bio.css";
+import Icon from "../Icon/Icon";
+import { Link } from "react-router-dom";
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faEdit } from "@fortawesome/free-solid-svg-icons";
+import InputTextField from "components/InputTextField/InputTextField";
+
+library.add(faEdit);
+// Class Based React Component
+class Bio extends Component {
+  constructor(props) {
+    super(props);
+    console.log(props);
+
+    // Default CSS class to apply to the Component
+    this.state = {
+      classList: "Bio",
+      isUser: this.props.isUser
+    };
+  }
+  // Runs after Component is loaded in the broswer
+  componentDidMount() {}
+
+  // Runs after a component has been updated
+  componentDidUpdate() {}
+
+  // Runs right before a component is removed from the DOM
+  componentWillUnmount() {}
+
+  render() {
+    console.log(this.props.isUser);
+    const isUser = this.state.isUser;
+
+    return (
+      <div className={this.state.classList}>
```

Comment:
No need to keep the class in state - can you just put the class here at the `className` attribute?